### PR TITLE
fix 404 in docker images blog

### PR DESCRIPTION
**Pull request by susanev**

### CODE REVIEW

File: themes/default/content/blog/introducing-new-docker-images/index.md [hunk 1]
```diff
@@ -238,7 +238,7 @@ To achieve this level of interoperability, however, we need to have some depende
...
 
-To provide a seamless experience for our users, our existing [pulumi/pulumi Docker image](https://github.com/pulumi/pulumi/tree/master/docker/pulumi) contains every possible dependency a user might need when provisioning cloud infrastructure. We install everything from the [Helm CLI](https://helm.sh/) to the [Azure CLI](https://docs.microsoft.com/en-us/cli/azure/get-started-with-azure-cli?view=azure-cli-latest) to ensure you can use the container for any Pulumi program. Unfortunately, as we began to support more cloud providers and more languages, the size of the image has grown considerably.
+To provide a seamless experience for our users, our existing [pulumi/pulumi Docker image](https://github.com/pulumi/pulumi-docker-containers) contains every possible dependency a user might need when provisioning cloud infrastructure. We install everything from the [Helm CLI](https://helm.sh/) to the [Azure CLI](https://docs.microsoft.com/en-us/cli/azure/get-started-with-azure-cli?view=azure-cli-latest) to ensure you can use the container for any Pulumi program. Unfortunately, as we began to support more cloud providers and more languages, the size of the image has grown considerably.
```
Comment (nimbinatus) [inline themes/default/content/blog/introducing-new-docker-images/index.md:241]: Did you want to point to the exact pulumi/pulumi image, or is it okay that it goes to the repo that has all of the images?

**Review by nimbinatus — APPROVED** [review]
Non-blocking question, but +1

File: themes/default/content/blog/introducing-new-docker-images/index.md (hunk 1 above)
Comment (susanev) [reply themes/default/content/blog/introducing-new-docker-images/index.md:241]: oh lol sry, will fix

Comment (susanev) [reply themes/default/content/blog/introducing-new-docker-images/index.md:241]: oh no, i dont think so. thats linked at the top of the blog.

**Pull request merged**